src/stores/AllEventsStore.test.js

```diff
@@ -0,0 +1,91 @@
+import { Routes } from 'constants';
+jest.mock('../network/graphql/queries'); // block and manually mock our backend
```

Comment:
move under imports?

Reply:
Initially I thought this has to be before the import module that uses backend, but now it seems unnecessary. Moved.